### ISSUE TITLE
Use upstream package?

### ISSUE THREAD

**Issue by drupol** [shivaraj-bh/ollama-flake#8]
Now that `open-webui` has been upstreamed, do you think this project should be updated to use it?

**Comment by shivaraj-bh** [comment]
Yes, it should be. Even better, I am thinking about moving the services here over to `services-flake`: https://github.com/juspay/services-flake/pull/219

Some of the benefits:
- One-place for all the services that run on Linux and macOS
- Can make use of the existing documentation site
- Multiple instance of a service (although it might not make sense for the services in `ollama-flake`)
- Leverage the existing testing infrastructure

I had only created this repo to manage the `open-webui` and other packages that would come in the future. Now that it has been upstreamed, I don’t see any reason not to move these services to `services-flake`.

**Comment by drupol** [comment]
Oh yes let's do it :)

**Comment by drupol** [comment]
Cool ! What's left to do then now?

**Comment by shivaraj-bh** [comment]
> Cool ! What's left to do then now?

open-webui and private-gpt. Will be upstreaming open-webui soon: https://github.com/juspay/services-flake/pull/219

**Comment by drupol** [comment]
But open-webui is done already. I don't understand.

**Comment by drupol** [comment]
Oh never mind. Ignore me.
